src/main/java/org/shemenev/v2/Record.java

```diff
@@ -44,17 +45,20 @@ public String getDescription() {
     }
 
 
-
-
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
Круто. Разобрался. Но можн чекнуть более коротко
```suggestion
        if (!(o instanceof Record)) return false;
```